client/components/theme/test/index.jsx

```diff
@@ -2,155 +2,124 @@
  * @jest-environment jsdom
  */
 import { parse } from 'url';
-import { shallow } from 'enzyme';
-import { createElement } from 'react';
-import ReactDom from 'react-dom';
-import TestUtils from 'react-dom/test-utils';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import { Theme } from '../';
 
 jest.mock( 'calypso/components/popover-menu', () => 'components--popover--menu' );
 jest.mock( 'calypso/components/popover-menu/item', () => 'components--popover--menu-item' );
 
 describe( 'Theme', () => {
-	let props;
-	let themeNode;
-
-	beforeEach( () => {
-		props = {
-			theme: {
-				id: 'twentyseventeen',
-				name: 'Twenty Seventeen',
-				screenshot:
-					'https://i0.wp.com/s0.wp.com/wp-content/themes/pub/twentyseventeen/screenshot.png?ssl=1',
-			},
-			buttonContents: { dummyAction: { label: 'Dummy action', action: jest.fn() } }, // TODO: test if called when clicked
-			translate: ( string ) => string,
-			setThemesBookmark: () => {},
-		};
-	} );
+	const props = {
+		theme: {
+			id: 'twentyseventeen',
+			name: 'Twenty Seventeen',
+			screenshot:
+				'https://i0.wp.com/s0.wp.com/wp-content/themes/pub/twentyseventeen/screenshot.png?ssl=1',
+		},
+		buttonContents: { dummyAction: { label: 'Dummy action', action: jest.fn() } }, // TODO: test if called when clicked
+		translate: ( string ) => string,
+		setThemesBookmark: () => {},
+		onScreenshotClick: () => {},
+	};
 
 	describe( 'rendering', () => {
 		describe( 'with default display buttonContents', () => {
-			beforeEach( () => {
-				props.onScreenshotClick = jest.fn();
-				const themeElement = TestUtils.renderIntoDocument( createElement( Theme, props ) );
-				themeNode = ReactDom.findDOMNode( themeElement );
-			} );
-
 			test( 'should render a <div> with a className of "theme"', () => {
-				expect( themeNode ).not.toBeNull();
-				expect( themeNode.nodeName ).toBe( 'DIV' );
-				expect( themeNode.className ).toEqual( expect.stringContaining( 'theme is-actionable' ) );
-				expect( themeNode.getElementsByTagName( 'h2' )[ 0 ].textContent ).toBe(
+				const { container } = render( <Theme { ...props } /> );
+				expect( container.firstChild ).toHaveClass( 'theme', 'is-actionable' );
+				expect( container.firstChild.nodeName ).toBe( 'DIV' );
```

Comment:
Is this (testing that it's a div) just testing an implementation detail?

Reply:
You're right, that shouldn't be relevant to the user. Fixed it via [768af82](https://github.com/Automattic/wp-calypso/pull/64186/commits/768af828728ecb0cde4128880e0c5590db35d7e0) 👍🏻